galaxy/main/migrations/0118_scoring_edits_remove_contentrule.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-09-25 19:35
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0117_namespace_search_index'),
```

Comment:
I added a migration named `0118_scoring_edits_remove_contentrule` in #1176 that has the same dependency as this. You'll need to rebase against devel and update this to reference the migration that I just added